ci-operator/step-registry/rhoai/deploy-dsc/rhoai-deploy-dsc-commands.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Deploying a DataScience Cluster"
+cat <<EOF | oc apply -f -
+apiVersion: datasciencecluster.opendatahub.io/v1
+kind: DataScienceCluster
+metadata:
+  name: "${DSC_NAME}"
+  labels:
+    app.kubernetes.io/name: datasciencecluster
+    app.kubernetes.io/instance: "${DSC_NAME}"
+    app.kubernetes.io/part-of: "${OPERATOR_NAME}"
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: "${OPERATOR_NAME}"
+spec:
+  components:
+    codeflare:
+      managementState: Managed
+    kserve:
+      managementState: Managed
+    trustyai:
+      managementState: Removed
+    ray:
+      managementState: Managed
+    kueue:
+      managementState: Managed
+    workbenches:
+      managementState: Managed
+    dashboard:
+      managementState: Managed
+    modelmeshserving:
+      managementState: Managed
+    datasciencepipelines:
+      managementState: Managed
+    trainingoperator:
+      managementState: Removed
+EOF
+
+echo "⏳ Wait for DataScientCluster to be deployed"
+oc wait --for=jsonpath='{.status.phase}'=Ready datasciencecluster/${DSC_NAME} --timeout=9000s 
+
+echo "Apply DSCInitialization CustomResource"
```

Comment:
the usual flow has DSCI deployed first, DSC later. Also, DSCI is automatically deployed by the operator.  We can still apply it ofc, but the question is:

do we want to apply it in order to a predefined configuration which might be different from the default one?

Reply:
@liswang89 Did we try running without manually creating the DSCI?
Also, I can see the DSC not being properly intercepted by the tests, see:
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/58481/rehearse-58481-periodic-ci-red-hat-data-services-ods-ci-master-rhoai2.13-ocp4.18-lp-interop-rhoai-interop-aws/1865163929022369792#1:build-log.txt%3A341